sources/lib/Model/ContainerModelLayerPooler.php

```diff
@@ -50,4 +50,15 @@ protected function createClient($class)
 
         return parent::createClient($class);
     }
+
+    /**
+     * @param PsrContainer|ContainerInterface $container
+     */
+    public function setContainer($container)
```

Comment:
You can type hint with the `PsrContainerInterface`, The `ContainerInterface` extends it so you can inject both without the if block below